prettier.json

```diff
@@ -0,0 +1,9 @@
+{
+	"bracketSpacing": false,
+	"endOfLine": "lf",
+	"jsxSingleQuote": true,
+	"singleQuote": true,
```

Comment:
Somehow, `"singleQuote": true` doesn't seem to have had any effect. I see the opposite in the code.

Reply:
Renaming `prettier.json` to  `.prettierrc` worked here, right now. I'm sending in a few minutes a commit updating this one and all affected files.